script.js

```diff
@@ -1,4 +1,24 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var results = scissorsPaperStone(input);
+  return results;
+};
+
+var scissorsPaperStone = function (input) {
+  var listOfChoice = ["scissors", "paper", "stone"];
+  var computerChoice = Math.floor(Math.random() * listOfChoice.length);
+  //var computerChoice = listOfChoice[randomChoice];
+  var userChoice = listOfChoice.indexOf(input);
+  var results = userChoice - computerChoice;
+  var winningCases = [-1, 2];
+  var losingCases = [1, -2];
```

Comment:
You can consider leaving comments for these statements so that anyone reading will be able to understand it easliy